packages/output/package.json

```
@@ -48,6 +49,9 @@
     "@googleforcreators/rich-text": "*",
     "@googleforcreators/units": "*",
     "@googleforcreators/url": "*",
+    "@googleforcreators/types": "*",
+    "@types/react": "^18.0.21",
+    "@types/react-dom": "^18.0.6",
```

We currently use React 17

```suggestion
    "@types/react": "^17.0.44",
    "@types/react-dom": "^17.017",
```